ovos_plugin_manager/templates/tts.py

```
@@ -1119,7 +1119,16 @@ class StreamingTTSCallbacks:
     def __init__(self, bus, play_args=None, tts_config=None):
         self.bus = bus
         self.config = tts_config or {}
-        self.play_args = play_args or ["paplay"]
+        if not play_args:
+            # Check for the available player depending on the system's audio server.
+            player = shutil.which("pw-play") or shutil.which("paplay") or shutil.which("ffplay")
```

Ruff (0.8.2)

Undefined name `shutil`

(F821)